src/common/meta/src/key/table_info.rs

```diff
@@ -52,6 +55,58 @@ impl TableInfoManager {
         Self { kv_backend }
     }
 
+    pub async fn get_old(&self, table_name: &TableName) -> Result<Option<TableInfoValue>> {
```

Comment:
What does the "old" mean? Is it used for compatibility purposes? Should we consider removing this code in the future? If that's the case, please add a TODO and comment.